docs/consuming_with_cmake.rst

```diff
@@ -0,0 +1,91 @@
+.. _consuming_with_cmake:
+
+Consuming with CMake
+==================== 
+
+The best way to use oneMKL is through integration with CMake. This can be done in two ways:
```

Comment:
IMO the first sentence here is not really true, certainly not for everyone. I would remove it.

Reply:
You're right - its hard to look beyond how I use it! Done in [931ee0f](https://github.com/oneapi-src/oneMKL/pull/510/commits/931ee0f68ff1bd61a5f9f7c1957559a4a41a2215).